src/docs/BUILD.md

```diff
@@ -37,6 +37,34 @@ The preferred approach is to build both OpenVINO and OpenVINO GenAI from sources
 
 ## Build Instructions
 
+### Build OpenVINO GenAI as OpenVINO Extra Module
+
+OpenVINO GenAI can be built as an extra module during the OpenVINO build process. This method simplifies the build process by integrating OpenVINO GenAI directly into the OpenVINO build.
+
+1. Clone OpenVINO repository:
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino.git
+    ```
+2. Clone OpenVINO GenAI repository:
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino_genai.git
+    ```
+3. Create a build directory:
+    ```sh
+    mkdir build
+    ```
```

Comment:
cmake seems to be fine if the folder is missing.
Don't forget to fix the numbers below

Reply:
Fixed